npm/webpack-dev-server/src/plugin.ts

```diff
@@ -64,7 +68,16 @@ export default class CypressCTOptionsPlugin {
 
         if (stats.hasErrors()) {
           this.errorEmitted = true
-          this.devServerEvents.emit('dev-server:compile:error', stats.toJson().errors?.[0])
+
+          // webpack 4: string[]
+          // webpack 5: Error[]
+          const errors = stats.toJson().errors as Array<Error | string>
+
+          if (!errors || !errors.length) {
+            return
+          }
```

Comment:
What does it mean if `hasErrors()` but there is no or an empty `errors` array?

Reply:
No errors, it seems - it's not entirely clear from the docs.